test/csuite/wt4803_cache_overflow_abort/main.c

```diff
@@ -0,0 +1,177 @@
+#include "test_util.h"
+
+#include <signal.h>
+#include <sys/wait.h>
+
+#define	NUM_KEYS	2000
+
+static int
+handle_message(WT_EVENT_HANDLER *handler, WT_SESSION *session, int error,
+    const char *message)
```

Comment:
Minor style: when breaking lines we make the 2nd line longer. I'd break after `*handler,`.

Reply:
98d8d9d